float64 long_v
float64 lat_v
float64 long_dis
float64 lat_dis
